In [1]:
import pandas as pd

In [12]:
population = pd.read_csv('total-population.csv')

print ("Estimated Total Population:",  population['Estimate; Total'].sum(axis=0))
print("")

sortedP = population.sort_values('Estimate; Total')
print (sortedP.head(3))
print ("")
print (sortedP.tail(3))

Estimated Total Population: 309327.0

   Neighborhood  Id  Estimate; Total  Margin of Error; Total
21      Chateau  22              3.0                3.000000
71  South Shore  72             12.0               14.000000
57  North Shore  58            205.0               66.753277

           Neighborhood  Id  Estimate; Total  Margin of Error; Total
13            Brookline  14          13160.0              943.065215
68            Shadyside  69          13562.0              861.984919
77  Squirrel Hill South  78          16042.0              889.712313


## Estimated Total Population: 309327 People

# The 3 most populated neighborhoods:

- Squirrel Hill South: 16042 People
- Shadyside: 13562 People
- Brookline: 13160 People 

# The 3 least populated neighborhoods:

- North Shore: 205 People
- South Shore: 12 People
- Chateau: 3 People

# Looking at Crime data compared to Population data:

In [28]:
#Crime Data
crime = pd.read_csv('crime.csv')

#Sorted Alphabetically
crime = crime.sort_values('INCIDENTNEIGHBORHOOD')['INCIDENTNEIGHBORHOOD'].value_counts(sort = False).rename_axis("Neighborhood")
print(crime.to_string())

Neighborhood
Allegheny Center              26
Allegheny West                 6
Allentown                     44
Arlington                     23
Arlington Heights              6
Banksville                    16
Bedford Dwellings             29
Beechview                     66
Beltzhoover                   15
Bloomfield                    52
Bluff                         51
Bon Air                       16
Brighton Heights              73
Brookline                     76
California-Kirkbride          17
Carrick                      118
Central Business District    210
Central Lawrenceville         29
Central North Side            28
Central Oakland               36
Chateau                       31
Crafton Heights               45
Crawford-Roberts              48
Duquesne Heights              14
East Allegheny                56
East Hills                    67
East Liberty                  91
Elliott                       24
Esplen                         3
Fairywood                     

# The number of crimes committed in the 3 most populated neighborhoods:

- Squirrel Hill South: 51 Cases
- Shadyside: 54 Cases
- Brookline: 76 Cases

# The number of crimes committed in the 3 least populated neighborhoods:

- North Shore: 23 Cases
- South Shore: 13 Cases
- Chateau: 31 Cases [^1]

[^1]: There seems to be a mistake with the population data or Crime data regarding Chateau